include/sycl/algorithm/mismatch.hpp

```diff
@@ -64,27 +64,23 @@ std::pair<ForwardIt1, ForwardIt2> mismatch(ExecutionPolicy& exec,
   const auto device = q.get_device();
 
   const auto length = std::min(size1, size2);
-  const auto local = std::min(
-      device.template get_info<cl::sycl::info::device::max_work_group_size>(),
-      length);
-  const auto global = exec.calculateGlobalSize(length, local);
+  auto ndRange = exec.calculateNdRange(length);
```

Comment:
If `ndRange` isn't modified, it needs to be `const`.

Reply:
This one is modified below